src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -675,6 +675,8 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration = new
 
         protected DynamoViewModel(StartConfiguration startConfiguration)
         {
+            Dispatcher.CurrentDispatcher.UnhandledException += CurrentDispatcher_UnhandledException;
```

Comment:
@pinzart90 does it make sense to also subscribe to the appdomain exception event here?

Reply:
I am not sure where we should put this handler. Theoretically it should be added as soon as possible in the application lifecycle so that more exceptions can be handled.
Also this handler will not catch exceptions coming from other threads. THat is where AppDomain.UNhandledException would come in handy. However it seems a bit risky since it could be called in non UI threads (at least that's what I;ve noticed in my testing)